code/game/gamemodes/objective.dm

```diff
@@ -347,6 +347,36 @@ datum/objective/hijack
 						return 0
 		return 1
 
+/datum/objective/hijackclone
+	explanation_text = "Hijack the emergency shuttle by ensuring only you (or your copies) escape."
+
+	check_completion()
+		if(!owner.current)
+			return 0
+		if(!emergency_shuttle.returned())
+			return 0
+
+		var/area/A = locate(/area/shuttle/escape/centcom)
+
+		for(var/mob/living/player in player_list) //Make sure nobody else is onboard
+			if(player.mind && player.mind != owner)
+				if(player.stat != DEAD)
+					if(istype(player, /mob/living/silicon))
+						continue
+					if(get_area(player) == A)
+						if(player.real_name != owner.current.real_name && !istype(get_turf(player.mind.current), /turf/simulated/shuttle/floor4))

```

Comment:
Not really sure what you mean it's checking that real name is identical--considering how difficult this is to change (and AFAIK, the only way that players can alter this is via genetics), I'm not really sure if that's a a problem. The floor check is to make an exception for the brig area on the shuttle.